packages/apps/spaces/ui/media/icons/LeftIndent.tsx

```diff
@@ -1,7 +1,7 @@
 import { Icon, IconProps } from '@ui/media/Icon';
 
 export const LeftIndent = (props: IconProps) => (
-  <Icon viewBox='0 0 24 24' fill='none' boxSize='4' {...props}>
+  <Icon viewBox='0 0 16 16' fill='none' boxSize='4' {...props}>
```

Comment:
Please re-gen icons after you revert the script script. This is potentially causing a lot of visual bugs in other places where those icons have been used.

Reply:
Actually it fixes the issues in other places